google-beta/resource_container_cluster.go

```diff
@@ -2721,7 +2721,10 @@ func flattenMasterAuth(ma *containerBeta.MasterAuth) []map[string]interface{} {
 
 func flattenClusterAutoscaling(a *containerBeta.ClusterAutoscaling) []map[string]interface{} {
 	r := make(map[string]interface{})
-	if a == nil || !a.EnableNodeAutoprovisioning {
+	if a == nil {
+		r["enabled"] = false
```

Comment:
I think this is great for now, as it won't be a breaking change.  I'll create a new issue for our next major release to change this to `nil`.

Reply:
ok!